ft-fine-tune/fine_tune_unittests.py

```diff
@@ -0,0 +1,132 @@
+                0,
+                2178473742,
+                'Mon Jun 15 07:45:14 PDT 2009',
+                'NO_QUERY',
+                'debsylou',
+                '@imBdW neither...5pm is more like 8pm ',
+            ],
+            [
+                4,
+                1882301030,
+                'Fri May 22 06:37:15 PDT 2009',
+                'NO_QUERY',
+                'JulzM',
+                '@crackbarbie This is my road kill http://twitpic.com/6m1q .. ',
+            ],
+            [
+                0,
+                2055350279,
+                'Sat Jun 06 09:09:23 PDT 2009',
+                'NO_QUERY',
+                'TreylinRae',
+                "Stopping to get food and a floatie for brodie and then on the way to raging waters in san dimas!! Woo! I'm so tired  haha",
+            ],
+        ]
+        self.df_columns = [
+            "target",
+            "timestamp",
+            "datetime",
+            "query",
+            "user",
+            "text",
+        ]
+        self.train_df = pd.DataFrame(self.train_data, columns=self.df_columns)
+        self.valid_df = pd.DataFrame(self.valid_data, columns=self.df_columns)
+
+        # Define dummy arguments
+        self.input_filename = "./train.csv"
+
+        # Define lists for testing map_labels
+        self.trainy = pd.Series([0, 4, 0, 0, 0])
+        self.validy = pd.Series([0, 4, 0])
+
+        # Define +/- labels
+        self.positive_label = "4"
+        self.neutral_label = "2"
+        self.negative_label = "0"
+
+        # Expected values
+        self.train_mapped = pd.core.series.Series([0, 2, 0, 0, 0])
+        self.valid_mapped = pd.core.series.Series([0, 2, 0])
+
+    def assertIsFile(self, path):
+        if not pl.Path(path).resolve().is_file():
+            raise AssertionError("File does not exist: %s" % str(path))
+
+    def test_data_paths(self):
+        """Checks if input file provided"""
+        file = pl.Path("./1.6m_twitts.csv")
+        self.assertIsFile(file)
+
+    def test_return_type(self):
+        """Checks if the function returns pd.core.series.Series"""
+        returned_lists_trainy = self.trainy.apply(lambda x: decode_sentiment(x))
+        returned_lists_validy = self.validy.apply(lambda x: decode_sentiment(x))
+        self.assertIsInstance(returned_lists_trainy, pd.core.series.Series)
+        self.assertIsInstance(returned_lists_validy, pd.core.series.Series)
+
+    def test_list_values(self):
+        """Checks if function returns list with just '2, 1, 0's"""
+        returned_lists_trainy = self.trainy.apply(lambda x: decode_sentiment(x))
+        # print("returned_lists_trainy", returned_lists_trainy)
+        returned_lists_validy = self.validy.apply(lambda x: decode_sentiment(x))
+        # print("returned_lists_validy", returned_lists_validy)
+        self.assertEqual(all(returned_lists_trainy), all(self.train_mapped))
+        self.assertEqual(all(returned_lists_validy), all(self.valid_mapped))
+
```

Comment:
fine_tune_unittests.py missing main:
```python:
if __name__ == "__main__":
    unittest.main()
```

Reply:
Sure, I will add it and re_submit it for approval!